pkg/controller/status/status.go

```diff
@@ -581,27 +585,24 @@ func (m *statusManager) isInitialized() bool {
 	return m.enabled != nil
 }
 
-// removeTigeraStatus returns true and removes the status displayed in TigeraStatus if corresponding CR not found
-func (m *statusManager) removeTigeraStatus() bool {
+// removeTigeraStatus status displayed in TigeraStatus if corresponding controller's CR is not found
```

Comment:
This comment doesn't make sense to me.

Reply:
well yeah, that's because the sentence doesn't make any sense, duh.